engine/runtime/src/main/scala/org/enso/compiler/SerializationManager.scala

```diff
@@ -146,7 +259,7 @@ class SerializationManager(compiler: Compiler) {
       abort(module)
       None
     } else {
-      while (isSerializing(module)) {
+      while (isSerializingModule(module)) {
         Thread.sleep(100)
```

Comment:
`wait(100)` and a `notifyAll()` when serialization is over would be a nicer that unconditional sleep.